app/controllers/settings/featured_tags_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Settings::FeaturedTagsController < Settings::BaseController
+  layout 'admin'
+
+  before_action :authenticate_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    @featured_tag   = FeaturedTag.new
+    @most_used_tags = Tag.most_used(current_account).where.not(id: @featured_tags.map(&:id)).limit(10)
+  end
+
+  def create
+    @featured_tag = FeaturedTag.new(featured_tag_params)
+    @featured_tag.account = current_account
+    @featured_tag.statuses_count = current_account.statuses.tagged_with(@featured_tag.tag).count
+    @featured_tag.last_status_at = current_account.statuses.tagged_with(@featured_tag.tag).maximum(:created_at)
+    @featured_tag.save
+
+    redirect_to settings_featured_tags_path
+  end
+
+  def destroy
+    @featured_tag.destroy
```

Comment:
Probably want `destroy` so you get an exception you can rescue from if it fails, yes?